Sources/PackageModel/SwiftSDKBundle.swift

```diff
@@ -202,47 +211,48 @@ public struct SwiftSDKBundle {
         _ fileSystem: some FileSystem,
         _ archiver: some Archiver
     ) async throws -> AbsolutePath {
-        let regex = try RegEx(pattern: "(.+\\.artifactbundle).*")
-
-        guard let bundleName = bundlePath.components.last else {
-            throw SwiftSDKError.invalidPathOrURL(bundlePath.pathString)
+        // If there's no archive extension on the bundle name, assuming it's not archived and returning the same path.
+        guard !bundlePath.pathString.hasSuffix(".artifactbundle") else {
```

Comment:
dont we have a constant for that somewhere?

Reply:
Yes, I've declares now `artifactBundleExtension` in `ArtifactsArchiveMetadata.swift`